manifests/package.pp

```diff
@@ -1,4 +1,10 @@
-class puppet::package {
+class puppet::package (
+  $agent_package  = $puppet::params::agent_package,
+  $master_package = $puppet::params::master_package,
```

Comment:
rename those to `$agent` and `$server` please for consistency with the class names and for not having to write the word package twice (`$puppet::package::agent` vs `$puppet::package::agent_package`)